client/blocks/plan-storage/style.scss

```diff
@@ -24,7 +24,7 @@
 }
 
 .plan-storage__storage-link {
-	margin-left: 3px;
+	margin-left: 12px;
```

Comment:
Just FYI I put `3px` keeping in mind that the test could increase in other languages. But it's rightly handled by the flexbox implementation, right?